### PR TITLE
[fix bug 1172375] Update Hello FTE redirect for non-Firefox users from /new to Hello product page

### PR DESCRIPTION
**Pull request by pmclanahan**

### CODE REVIEW

**Comment by pmclanahan** [comment]
@alexgibson this is my rework of your commit from #3051. Let me know what you think.

**Comment by alexgibson** [comment]
Thanks @pmclanahan - I agree this is the simpler solution for now. Tested locally and works as expected, thanks for the assist! :cake:

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -414,16 +417,18 @@ def redirect_to(self):
...
         query = self.request.META.get('QUERY_STRING')
         query = '?' + query if query else ''
 
         user_agent = self.request.META.get('HTTP_USER_AGENT', '')
         if 'Firefox' not in user_agent:
-            return reverse('firefox.new') + query
+            return reverse(self.non_fx_redirect) + query
             # TODO : Where to redirect bug 757206
 
+        return None
```
Comment (jgmize) [inline bedrock/firefox/views.py]: NIT: I personally prefer to leave this as implicit, but to each their own.

Comment (pmclanahan) [inline bedrock/firefox/views.py]: It was implicit, but agibson had made his explicit, then the Zen crept into my head, and this is much more obvious... to each their own I guess. :guardsman:

**Pull request merged**